ci-operator/config/openshift/loki/openshift-loki-release-6.0.yaml

```diff
@@ -0,0 +1,161 @@
+base_images:
+  base:
+    name: "4.16"
+    namespace: ocp
+    tag: base-rhel9
+  loki-tag:
+    name: v3.1.0
+    namespace: logging
+    tag: loki
+  lokistack-gateway:
+    name: mirror-5.y
+    namespace: logging
+    tag: lokistack-gateway
+  ocp_builder_rhel-9-golang-1.21-builder-multi-openshift-4.16:
+    name: builder
+    namespace: ocp
+    tag: rhel-9-golang-1.21-builder-multi-openshift-4.16
+  opa-openshift:
+    name: mirror-5.y
```

Comment:
Shouldn't we also edit `core-services/image-mirroring/_config.yaml` to add mirror-6.y?

Reply:
Maybe `mirror-main` would be "more true"? Would then make sense in both versions.

Reply:
Yes this is best served in separate cleanup PR once we get the branch config in.